Jenkinsfile

```
@@ -86,6 +86,11 @@ def checkGeneratedSchemasAreUpToDate(govuk) {
     }
 }
 
+boolean schemasDirChangedInPr(branchName) {
```

I'd recommend naming this `schemasDirChangedInBranch` instead of `schemasDirChangedInPr` as we run a CI build each time a branch is pushed, even if there isn't a PR.
```suggestion
boolean schemasDirChangedInBranch(branchName) {
```